lucene/lucene-directory/src/main/java/org/infinispan/lucene/impl/CommonLockObtainUtils.java

```diff
@@ -0,0 +1,36 @@
+package org.infinispan.lucene.impl;
+
+import java.io.IOException;
+
+import org.apache.lucene.store.LockObtainFailedException;
+
+public class CommonLockObtainUtils {
+
+   private static int MAX_LOCK_ACQUIRE_MILLISECONDS = 10;
+
+   private CommonLockObtainUtils() {
+      //not to be constructed
+   }
+
+   public static void attemptObtain(ObtainableLock lock) throws IOException {
+      int attempts = 0;
+      while (lock.obtain()==false) {
```

Comment:
I generally think that `!` is not very visible :)
Which is why on Hibernate we mandate whitespaces around it. Feel free to change, or shall I?